app/gui/src/project-view/components/visualizations/JSONVisualization/JSONVisualization.vue

```diff
@@ -63,12 +63,19 @@ function createProjection(path: (string | number)[][]) {
     ),
   )
 }
+
+function stopCopy(event: KeyboardEvent) {
+  if (event.ctrlKey && event.code === 'KeyC') {
```

Comment:
This does not work on Mac because the copy uses the Meta key. Also, on a non-QWERTY keyboard layout, copy can be a different key code as well.